pkg/apis/authorization/v1beta1/register.go

```diff
@@ -37,10 +39,15 @@ func AddToScheme(scheme *runtime.Scheme) {
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) {
 	scheme.AddKnownTypes(SchemeGroupVersion,
+		&v1.ListOptions{},
```

Comment:
> MethodNotSupported (405 I think).

It definitely fails with a MethodNotSupported, I just don't remember the exact http response code.